src/interactions/components/handle.rs

```diff
@@ -3,9 +3,8 @@ use crate::interactions::context::ComponentCtx;
 use super::dismiss::handle_dismiss;
 
 pub async fn handle_component(ctx: ComponentCtx) -> anyhow::Result<()> {
-    match ctx.data.custom_id.as_str() {
-        "stateless::dismiss_notification" => handle_dismiss(&ctx).await?,
-        _ => {}
+    if let "stateless::dismiss_notification" = ctx.data.custom_id.as_str() {
+        handle_dismiss(&ctx).await?
```

Comment:
keep the match here. There will be more options later, and the match will make it cleaner.

Reply:
oki